packages/portfolio-contract/src/plan-solve.ts

```diff
@@ -426,36 +441,80 @@ export const rebalanceMinCostFlowSteps = (
     lastChain = chosen.srcChain;
   }
 
-  const steps: MovementDesc[] = prioritized.map(({ edge, flow }) => {
-    Number.isSafeInteger(flow) ||
-      Fail`flow ${flow} for edge ${edge} is not a safe integer`;
-    const amount = AmountMath.make(graph.brand, BigInt(flow));
-
-    let details = {};
-    switch (edge.feeMode) {
-      case 'gmpTransfer':
-        // TODO: Rather than hard-code, derive from Axelar `estimateGasFee`.
-        // https://docs.axelar.dev/dev/axelarjs-sdk/axelar-query-api#estimategasfee
-        details = { fee: AmountMath.make(graph.feeBrand, 30_000_000n) };
-        break;
-      case 'gmpCall':
-        // TODO: Rather than hard-code, derive from Axelar `estimateGasFee`.
-        // https://docs.axelar.dev/dev/axelarjs-sdk/axelar-query-api#estimategasfee
-        details = { fee: AmountMath.make(graph.feeBrand, 30_000_000n) };
-        break;
-      case 'toUSDN': {
-        // NOTE USDN transfer incurs a fee on output amount in basis points
-        const usdnOut =
-          (BigInt(flow) * (10000n - BigInt(edge.variableFee))) / 10000n;
-        details = { detail: { usdnOut } };
-        break;
+  const padding = 3n; // 300% padding on gas estimates
```

Comment:
Reiterating https://github.com/Agoric/agoric-sdk/pull/11953#discussion_r2379675968 :
```suggestion
  /**
   * Pad each fee estimate in case the landscape changes between estimation and
   * execution.
   */
  const padFeeEstimate = (estimate: bigint): bigint => estimate * 3n;
```